javascript/javascript-1.md

```diff
@@ -105,3 +106,10 @@ Here are some other ways you can extend your calculator. Choose whichever one(s)
 
 ### Check for Understanding
 
+ Form small groups and discuss:
+ 
+- Mention some rules for declaring variables in JavaScript?
+- Mention some math operations that can be performed on number variables in JavaScript?
+- How to define and call a function in JavaScript?
+- How to find the length of a string?
+- Index of a string in JavaScript begins from?
```

Comment:
Are you asking about the first index being 0, @raibove ?

You can say:
"What is the first index of a string?"